parsec/core/fs/userfs/userfs.py

```diff
@@ -283,21 +289,26 @@ async def workspace_storage_task(
 
     async def _instantiate_workspace(self, workspace_id: EntryID) -> WorkspaceFS:
         # Workspace entry can change at any time, so we provide a way for
-        # WorskpaeFS to load it each time it is needed
+        # WorkspaceFS to load it each time it is needed
         def get_workspace_entry() -> WorkspaceEntry:
             user_manifest = self.get_user_manifest()
             workspace_entry = user_manifest.get_workspace_entry(workspace_id)
             if not workspace_entry:
                 raise FSWorkspaceNotFoundError(f"Unknown workspace `{workspace_id}`")
             return workspace_entry
 
+        async def get_previous_workspace_entry() -> WorkspaceEntry:
```

Comment:
Would be good to add a small docstring explaining we are retrieving previous encryption revision version of the workspace entry

Reply:
5769bda